From 25b1fb956cc8029d5030d93cf48faed823778c5e Mon Sep 17 00:00:00 2001
From: Martin Babinsky <mbabinsk@redhat.com>
Date: Mon, 25 Jul 2016 14:05:08 +0200
Subject: [PATCH] baseldap: Fix MidairCollision instantiation during entry
 modification

https://fedorahosted.org/freeipa/ticket/6097

Reviewed-By: Alexander Bokovoy <abokovoy@redhat.com>
---
 ipaserver/plugins/baseldap.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ipaserver/plugins/baseldap.py b/ipaserver/plugins/baseldap.py
index 6107e43a6ee17d9b9a63d9dc109664d8b232069f..f7844e3e7c59c259b9c8367d135b2dbefc3f0016 100644
--- a/ipaserver/plugins/baseldap.py
+++ b/ipaserver/plugins/baseldap.py
@@ -1466,7 +1466,7 @@ class LDAPUpdate(LDAPQuery, crud.Update):
                 entry_attrs.dn, attrs_list)
         except errors.NotFound:
             raise errors.MidairCollision(
-                format=_('the entry was deleted while being modified')
+                message=_('the entry was deleted while being modified')
             )
 
         self.obj.get_indirect_members(entry_attrs, attrs_list)
@@ -2344,7 +2344,7 @@ class BaseLDAPModAttribute(LDAPQuery):
                 entry_attrs.dn, attrs_list)
         except errors.NotFound:
             raise errors.MidairCollision(
-                format=_('the entry was deleted while being modified')
+                message=_('the entry was deleted while being modified')
             )
 
         for callback in self.get_callbacks('post'):
-- 
2.7.4

